### PR TITLE
Issue #49, "Programmatic Control of Popup"

### PR DESCRIPTION
**Pull request by lieszkol**
Please see Issue #49

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/DataGridExtensions?pullRequest=50) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/DataGridExtensions?pullRequest=50) before we can accept your contribution.<br/><br/>:x: lieszkol [sign now](https://cla.dotnetfoundation.org/dotnet/DataGridExtensions?pullRequest=50)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/DataGridExtensions?pullRequest=50) it.</sub>

**Review by tom-englert — CHANGES_REQUESTED** [review]

File: src/DataGridExtensions/DataGridFilterColumnControl.cs [hunk 1]
```diff
@@ -131,6 +135,23 @@ private void DataGrid_CollectionChanged(object? sender, NotifyCollectionChangedE
         public static readonly DependencyProperty FilterProperty =
             DependencyProperty.Register("Filter", typeof(object), typeof(DataGridFilterColumnControl));
 
+        /// <summary>
+        /// If the template includes a popup, the visibility of the popup can be controlled via this property.
+        /// </summary>
+        public bool IsPopupVisible
```
[the remark this answers is not in the document]
Comment (lieszkol) [reply src/DataGridExtensions/DataGridFilterColumnControl.cs:141]: And then do something like this in the viewmodel:

```
            if (Column5FilterColumnControl.FilterControl is FilterWithPopupControl filterWithPopupControl)
            {
                filterWithPopupControl.IsPopupVisible = true;
            }
```

?

I think it's more cumbersome for the end-user than writing `Column5FilterColumnControl.IsPopupVisible = true` but it works, I will make the required changes and update the PR.

File: src/DataGridExtensionsSample/Controls/FilterWithPopupControl.xaml.cs [hunk 2]
```diff
@@ -1,19 +1,30 @@
...
     /// </summary>
-    public partial class FilterWithPopupControl
+    public partial class FilterWithPopupControl: INotifyPropertyChanged
     {
         public FilterWithPopupControl()
         {
             InitializeComponent();
         }
 
+        private string caption;
```
[the remark this answers is not in the document]
Comment (lieszkol) [reply src/DataGridExtensionsSample/Controls/FilterWithPopupControl.xaml.cs:17]: Ok, I will make the changes and update the PR.

**Review by tom-englert — APPROVED** [review]

**Comment by tom-englert** [comment]
👍 😄

**Pull request merged**